tajo-core/src/main/java/org/apache/tajo/engine/parser/SQLAnalyzer.java

```diff
@@ -491,14 +506,16 @@ public WindowSpec buildWindowSpec(SQLParser.Window_specificationContext ctx) {
     WindowFrameStartBoundType boundType = null;
     if (checkIfExist(context.UNBOUNDED())) {
       boundType = WindowFrameStartBoundType.UNBOUNDED_PRECEDING;
-    } else if (checkIfExist(context.unsigned_value_specification())) {
+    } else if (checkIfExist(context.PRECEDING())) {
       boundType = WindowFrameStartBoundType.PRECEDING;
+    } else if (checkIfExist(context.FOLLOWING())) {
+      boundType = WindowFrameStartBoundType.FOLLOWING;
     } else {
       boundType = WindowFrameStartBoundType.CURRENT_ROW;
```

Comment:
I'm a little confused about the code because it passes several weeks after writing the code.
Sorry, forget about above comments.

Code in SQLAnalyzer.java only works when the user explicitly write the window frame specification on sql string, while ExprAnnotator::visitWindowFunction fills implicit information when user does not specify it.